projects/js-packages/components/components/boost-score-graph/style-tooltip.scss

```
@@ -43,6 +43,7 @@ $white: #fff;
     &__date {
         font-size: 1em;
         font-weight: 600;
+				// stylelint-disable-next-line declaration-property-unit-allowed-list -- this should be changed to a unitless value: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#values
```

What's up with the excessive indent here?
```suggestion
        font-weight: 600;
        // stylelint-disable-next-line declaration-property-unit-allowed-list -- this should be changed to a unitless value: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#values
```